ramls/instances-batch-response.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Response schema for instances batch request",
+  "type": "object",
+  "properties": {
+    "instances": {
+      "id": "instancesList",
+      "description": "List of Instances",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "$ref": "instance.json"
+      }
+    },
+    "errorMessages": {
+      "id": "errorMessagesList",
+      "description": "List of error messages",
+      "type": "array",
+      "items": {
+        "type": "string"
```

Comment:
Why is this an array of strings, instead of using the standard validation error structure? Or is this intended to convey other kinds of errors as well?

Reply:
You are right. This is an array of strings, each string is an error message for that Instance from collection, that can not be saved by some reason (validation, unprocessable entity). Of course this is a temporal solution before we decide how to report errors.